src/main/java/reposense/git/GitUtil.java

```diff
@@ -79,15 +85,60 @@ static String convertToGitFormatsArgs(List<Format> formats) {
 
     /**
      * Returns the {@code String} command to specify the globs to exclude for `git log` command.
+     * Also checks that every file in {@code String} file is within the given repository {@code File} root
```

Comment:
When citing the parameters, can just use the parameter name e.g. `{@code ignoreGlobList}`.
The `{@code String}` in the above header was used to described the return value as there was only a given type.